docs/r2/tensorboard_quickstart.ipynb

```diff
@@ -0,0 +1,463 @@
+{
```

Comment:
<p>If you do shift to tfds, you don't have to specify the steps_per_epoch parameter in model.fit() :)</p><p><br></p><p>```</p><p>model.fit(mnist_train, epochs=5, validation_data=mnist_test, callbacks=[tensorboard_callback])</p><p>```</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/tensorboard/pull/1870/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>